test/functional/admin_controller_test.rb

```diff
@@ -321,7 +321,7 @@ def teardown
     assert_equal 0, revision.status
     assert_equal 0, revision.author.status
     assert_not_equal node(:spam_targeted_page).latest.vid, revision.vid
-    assert_redirected_to "/wiki/revisions/<%= @node.slug %>" + '?_=' + Time.now.to_i.to_s
+    assert_redirected_to "/wiki/revisions/" + @revision.drupal_node.slug_from_path + '?_=' + Time.now.to_i.to_s
```

Comment:
OK -- while in the controller, we used an instance variable (starting with `@`) in the test, we are just using a local variable, so no `@` required. Check each changed line to ensure we're using what's available there -- either `@node` or `@revision` or `node` or `revision` -- and match the usage in that context.